src/Block/Service/AbstractBlockService.php

```diff
@@ -48,31 +48,6 @@ public function renderResponse(string $view, array $parameters = [], ?Response $
         return $response;
     }
 
-    /**
-     * Returns a Response object that cannot be cacheable, this must be used if the Response is related to the user.
-     * A good solution to make the page cacheable is to configure the block to be cached with javascript ...
-     *
-     * @deprecated since sonata-project/block-bundle 4.x and will be removed in 5.0.
-     *
-     * NEXT_MAJOR: remove
-     *
-     * @param array<string, mixed> $parameters
-     */
-    public function renderPrivateResponse(string $view, array $parameters = [], ?Response $response = null): Response
```

Comment:
this is currently called on SonataAdmin. Will also look into making that compatible with 5.x.

Reply:
Yeah would be great to have sonataAdmin 4 compatible with both SonataBlock 4 and 5.

Reply:
See https://github.com/sonata-project/SonataAdminBundle/pull/7792

will also look into allowing 5.0 via composer on SonataAdmin